OpenRA.Mods.RA/Lint/CheckActorReferences.cs

```diff
@@ -65,5 +65,4 @@ string[] GetFieldValues(ITraitInfo traitInfo, FieldInfo fieldInfo)
 						.F(actorInfo.Name, traitInfo.GetType().Name, fieldInfo.Name, type, v));
 		}
 	}
-}
-
+}
```

Comment:
I could but this change had a reason:

SA1507: The code must not contain multiple blank lines in a row.
SA1518: The code must not contain blank lines at the end of the file.